etrago/appl.py

```diff
@@ -85,7 +86,7 @@
         'max_iter': 100, # affects clustering algorithm, only change when neccesary
         'tol': 1e-6, # affects clustering algorithm, only change when neccesary
         'n_jobs': -1}, # affects clustering algorithm, only change when neccesary
```

Comment:
In a new environment, the clustering is not working because the `n_jobs` parameter was removed in scikit learn in  versions `>0.23`. We can either restrict the package version or remove the parameter from our functions. 
I have the tendency to use the second option to avoid new conflicts of package versions. What do you think?
@CarlosEpia: It is the same for clustering the electricity grid, we should agree one one solution

Reply:
I think the best option is to remove the n_jobs parameter. I will do it in my branch #361